neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/http/dialect/impl/HttpNeo4jTupleSnapshot.java

```diff
@@ -85,27 +73,12 @@ private Object readPropertyOnOtherNode(String column) {
 		String associationrole = rolesByColumn.get( column );
 		Node associatedEntity = toOneEntities.get( associationrole );
 		if ( associatedEntity == null ) {
-			// Not cached, let's look for it
-			associatedEntity = queries.findAssociatedEntity( neo4jClient, txId, keyValues(), associationrole );
-			if ( associatedEntity == null ) {
-				return null;
-			}
-			else {
-				toOneEntities.put( associationrole, associatedEntity );
-			}
+			return null;
```

Comment:
So, if I understand this change correctly, we now always get the associated entities?